web/src/content/posts.ts

```diff
@@ -0,0 +1,143 @@
+    tags: ['documentation', 'rsc', 'example-app'],
+    publishedAt: new Date('2024-04-23T12:00:00.000Z'),
+    author: authors.RobCameron,
+  },
+  {
+    slug: 'middleware-in-redwoodjs',
+    title: 'Middleware in RedwoodJS',
+    brief: 'An introduction to the new middleware in RedwoodJS',
+    tags: ['middleware', 'redwoodjs'],
+    publishedAt: new Date('2024-05-10T12:00:00.000Z'),
+    author: authors.DannyChoudhury,
+  },
+  {
+    slug: 'new-feature-ogimage-middleware',
+    title: 'New Feature: og:image Middleware',
+    brief:
+      '💡 og:image Middleware is only available in the latest canaries of Redwood!',
+    tags: ['seo', 'reactjs', 'middleware', 'redwoodjs'],
+    publishedAt: new Date('2024-05-13T12:00:00.000Z'),
+    author: authors.RobCameron,
+  },
+  {
+    slug: 'using-middleware-rss-and-sitemap',
+    title: 'Using Middleware: RSS and Sitemap',
+    brief:
+      'We recently redesigned our website for the new "Bighorn" epoch of Redwood. This included starting this series of blog posts which had a fun side effect of giving us an exercise to solve. How do we implement an RSS feed when our blog is built into a R...',
+    tags: ['javascript', 'middleware', 'redwoodjs'],
+    publishedAt: new Date('2024-05-21T12:00:00.000Z'),
+    author: authors.JoshGMWalker,
+  },
+  {
+    slug: 'experiments-rails-like-form-helpers',
+    title: 'Experiments: Rails-like Form Helpers',
+    brief:
+      'After ReactConf 2024 I was inspired by this tweet: https://x.com/chantastic/status/1791531154212033004 In it, Sam Selikoff says that the JS ecosystem has yet to demonstrate the same kind of powerful forms that have been available in Rails since jus...',
+    tags: ['forms', 'reactjs', 'rsc'],
+    publishedAt: new Date('2024-06-04T12:00:00.000Z'),
+    author: authors.RobCameron,
+  },
+  {
+    slug: 'love-reloaded-a-dx-story',
+    title: '"Love reloaded": A DX Story',
+    brief:
+      "Redwood aims is to provide a fast, robust and comprehensible React Server Component experience. We're iterating towards improving that experience by adding a RSC development server that supports live reload. What is Live Reload? Live reload is a feat...",
+    tags: ['web-development', 'dx', 'developer-experience', 'redwoodjs'],
+    publishedAt: new Date('2024-06-18T12:00:00.000Z'),
+    author: authors.PeterPistorius,
+  },
+  {
+    slug: 'moving-our-site-from-netlify-to-flyio',
+    title: 'Moving our site from Netlify to Fly.io',
+    brief:
+      "We recently switched our main website from Netlify to Fly.io. This was a pretty smooth process and we're happy with the results. Here's a quick overview of our experience.",
+    tags: [
+      'hosting',
+      'javascript',
+      'javascript-framework',
+      'redwoodjs',
+      'flyio',
+    ],
+    publishedAt: new Date('2024-07-02T12:00:00.000Z'),
+    author: authors.JoshGMWalker,
+  },
+]
+
+export function getPosts(): Post[] {
+  return posts
+    .filter((post) => {
```

Comment:
Once we switch to RSC then we can have proper unpublished articles but for now they're all included in the build they just get filtered on the browser right now.